README.md

```diff
@@ -1,213 +1,3 @@
-    - Added some extra stats to the Wins Per Day popup
-    - Added option in settings to show / hide Wins info for overlay
-  - 1.73
-    - Added options to settings screen for overlay color and flip to make it more visible to the user
-    - Added ability to manually resize overlay from the corners
-  - 1.72
-    - Changed overlay so it stays visible when you minimize amin screen
-  - 1.71
-    - Changed main screen to show Fastest / Longest qualifications for each level
-    - Fixed minor sorting issue in the grids
-  - 1.70
-    - Cleaned up auto update feature a bit
-  - 1.69
-    - Program will save last location of main window now and restore it when opened again
-  - 1.68
-    - Fixed Week / Day filters
-    - Added more filter options in settings
-    - Added logic to account for new levels that may come up in Season 2
-    - Added option to auto update program in settings
-  - 1.67
-    - Fixed times in database to be stored correctly as utc
-  - 1.66
-    - Hopefully fixed an issue with times counting down on overlay in rare cases
-  - 1.65
-    - Added export to MarkDown option
-    - Added ability to click on Win% label to toggle columns to show %s and back
-  - 1.64
-    - Fixed time issue when parsing log
-  - 1.63
-    - Added export options for both Html and BBCode when right clicking any grid
-  - 1.62
-    - Fixed some logic when deleting shows while filtered
-    - Switched the Longest/Fastest to align with Qualify/Gold
-  - 1.61
-    - Added logic to reset overlay position if it ended up off screen
-    - Tightened up the overlay when hiding the round info
-  - 1.60
-    - Added option to show tab on overlay for current filter
-  - 1.59
-    - Try and make sure deleted shows dont get added back accidentally
-  - 1.58
-    - Fixed rare case when deleting show didnt work
-  - 1.57
-    - Fixed overlay missing image on startup
-  - 1.56
-    - Add ability to show / hide information on overlay
-  - 1.55
-    - Fixed overlay getting out of wack if you change filters a lot
-  - 1.54
-    - Added mouse hover tool tip on Qualified / Gold / Silver / Bronze to show value as a %
-  - 1.53
-    - Fixed Filters on overlay not taking into account UTC time
-  - 1.52
-    - Fixed Time display on overlay not updating sometimes
-  - 1.51
-    - Fixed an issue around results coming from a previous match
-  - 1.50
-    - Fixed accidental debug typo
-  - 1.49
-    - Added filter options to settings page for overlay display
-  - 1.48
-    - Fixed Gold statistic on overlay, was using wrong medal type
-  - 1.47
-    - Added Gold statistic to overlay that rotates with Qualify
-  - 1.46
-    - Fixed overlay display not updating
-  - 1.45
-    - Cleaned up labels on overlay
-  - 1.44
-    - Fixed end of round numbers on overlay
-  - 1.43
-    - Added ability to delete Shows in Shows screen
\ No newline at end of file
+Please head to https://github.com/ShootMe/FallGuysStats for further information on this project.
```

Comment:
Looks like you've removed all the readme.md which is currently displayed at the bottom of the project. 

May need to adjust this so the log is still there.

Reply:
yes, my bad :(